doc/api/os.md

```diff
@@ -202,6 +202,8 @@ added: v2.3.0
 
 The `os.homedir()` method returns the home directory of the current user as a
 string.
+Calling this might rarely result in an exception due to e.g. `ENOENT` and other
```

Comment:
I don't think `ENOENT` should be returned. The errors returned by `os.homedir()` and the underlying `uv_os_homedir()` are things like `ENOBUFS`, `EINVAL`, and `ENOMEM`. I don't think we need to document this behavior, as we would need to add the same documentation to just about every call to libuv.